release-next.md

```diff
@@ -0,0 +1,48 @@
+# [NimbusML](https://docs.microsoft.com/en-us/nimbusml/overview) Next
+
+## **New Features**
+
+- **Time Series Spike And Change Point Detection**
+
+   [PR#135](https://github.com/microsoft/NimbusML/pull/135) added support
+   for time series spike and change point detection using the Independent
+   and identically distributed (IID) and Singular Spectrum Analysis (SSA)
+   algorithms.
+
+   **Spike Detection Examples**
+
+   - [IID using a pandas Series](https://github.com/microsoft/NimbusML/blob/3c689c63fefbaf0e6592ac3d427d474d53877dc3/src/python/nimbusml/examples/examples_from_dataframe/IidSpikeDetector_df.py)
```

Comment:
remve blob/3c689c63fefbaf0e6592ac3d427d474d53877dc

Reply:
I tried removing `blob/3c...` completely but then the link returns a 404. I switched out the `blob/3c...` with `tree/master` so that it points to the latest revision of the file on the master branch.